include/aie/Dialect/AIE/IR/AIEOps.td

```diff
@@ -820,7 +842,9 @@ def AIE_DMABDOp: AIE_Op<"dma_bd", []> {
 
   let arguments = (
     ins AnyMemRef:$buffer,
-        OptionalAttr<AIEI32Attr>:$offset,
+        // in words (multiples of element width) not bytes
```

Comment:
This changed from `multiples of 32b` (which I found straightforward) to `multiples of element width` (which I find a bit confusing). What was the reason for the change? Do we always assume that the element width is 32b?

Reply:
No - the arithmetic is done so that if you write `aie.dma_bd(%buf33_0 : memref<256xi16>, 0, 32)` then 32*2B = 64 bytes will transferred.

Reply:
So the length is always expressing the number of elements. Ok!